tests/test_object_states.py

```diff
@@ -695,3 +696,66 @@ def test_toggled_on():
     # Setter should work
     assert stove.states[ToggledOn].set_value(False)
     assert not stove.states[ToggledOn].get_value()
+
+@og_test
+def test_attached_to():
+    shelf_back_panel = og.sim.scene.object_registry("name", "shelf_back_panel")
+    shelf_shelf = og.sim.scene.object_registry("name", "shelf_shelf")
+    shelf_baseboard = og.sim.scene.object_registry("name", "shelf_baseboard")
+
+    shelf_back_panel.set_position_orientation([0, 0, 0.01], [0, 0, 0, 1])
+    shelf_back_panel.keep_still()
+    shelf_shelf.set_position_orientation([0, 0.03, 0.17], [0, 0, 0, 1])
+    shelf_shelf.keep_still()
+
+    # The shelf should not be attached to the back panel (no contact yet)
+    assert not shelf_shelf.states[AttachedTo].get_value(shelf_back_panel)
+
+    # Let the shelf fall
+    for _ in range(10):
+        og.sim.step()
+
+    # The shelf should be attached to the back panel
+    assert shelf_shelf.states[AttachedTo].get_value(shelf_back_panel)
+
+    assert shelf_shelf.states[AttachedTo].set_value(shelf_back_panel, True)
+    # The shelf should still be attached to the back panel
+    assert shelf_shelf.states[AttachedTo].get_value(shelf_back_panel)
+
+    assert shelf_shelf.states[AttachedTo].set_value(shelf_back_panel, False)
+    # The shelf should not be attached to the back panel
+    assert not shelf_shelf.states[AttachedTo].get_value(shelf_back_panel)
+
+    assert shelf_shelf.states[AttachedTo].set_value(shelf_back_panel, True)
+    # shelf should be attached to the back panel
+    assert shelf_shelf.states[AttachedTo].get_value(shelf_back_panel)
+
+    force_dir = np.array([0, 0, 1])
+    # A small force will not break the attachment
+    force_mag = 10
+    apply_force_at_pos(shelf_shelf.root_link, force_dir * force_mag, shelf_shelf.get_position())
+    og.sim.step()
+    assert shelf_shelf.states[AttachedTo].get_value(shelf_back_panel)
+
+    # A large force will break the attachment
+    force_mag = 1000
```

Comment:
was the force - breakage correlation issue ever resolved?

Reply:
no :(